modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/segregated/mixture_physics_diff.cmp

```diff
@@ -0,0 +1,12 @@
+COORDINATES absolute 1.e-6    # min separation not calculated
+
+TIME STEPS relative 1e-8
+
+# No GLOBAL VARIABLES
+
+ELEMENT VARIABLES
+	vel_x       relative 2e-1
+	vel_y       relative 5e-1
+	pressure    relative 6e-2
+	phase_2     relative 1e-2
+	phase_1_out relative 1e-2
```

Comment:
Could you just gold them separately? Obviously the two are not the same.

Reply:
1% on the phase over the whole domain is pretty good?